lms/djangoapps/grades/signals/handlers.py

```diff
@@ -70,51 +65,30 @@ def submissions_score_reset_handler(sender, **kwargs):  # pylint: disable=unused
     course_id = kwargs['course_id']
     usage_id = kwargs['item_id']
     user = user_by_anonymous_id(kwargs['anonymous_user_id'])
+    if user is None:
+        return
 
     SCORE_CHANGED.send(
         sender=None,
         points_possible=0,
         points_earned=0,
-        user=user,
+        user_id=user.id,
         course_id=course_id,
         usage_id=usage_id
     )
 
 
 @receiver(SCORE_CHANGED)
-def recalculate_subsection_grade_handler(sender, **kwargs):  # pylint: disable=unused-argument
+def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Generalize this name to `enqueue_grade_update` since we'll be updating both subsection and course grades asynchronously.